wandb/sdk/internal/system/assets/gpu_amd.py

```diff
@@ -30,10 +30,10 @@
 
 def get_rocm_smi_stats() -> Dict[str, Any]:
     command = [str(ROCM_SMI_CMD), "-a", "--json"]
-    output = (
-        subprocess.check_output(command, universal_newlines=True).strip().split("\n")
-    )[0]
-    return json.loads(output)  # type: ignore
+    output = subprocess.check_output(command, universal_newlines=True).strip()
+    if "No AMD GPUs specified" in output:
```

Comment:
lol

Reply:
yea, not the most elegant thing in the world. this will be gone when we switch to a native go lib